src/pages/Main/Main.scss

```
@@ -0,0 +1,208 @@
+/* stylelint-disable selector-class-pattern */
+body {
+  background-color: #fcf9ee;
+
+  .promoVideo {
+    position: relative;
+
+    .backgroundVid {
+      position: relative; /* 핵심코드 */
+      z-index: -1;
+      top: 20;
+      right: 0;
+      left: 0;
+      width: 100%;
+      height: 860px;
+      background-size: cover; /* 핵심코드 */
+      object-fit: cover;
+    }
+  }
+
+  .collectionWrap {
+    position: absolute; /* 핵심코드 */
+    top: 20%; /* 핵심코드 */
+    left: 50%; /* 핵심코드 */
+    transform: translate(-50%, -50%); /* 핵심코드 */
+    text-align: center;
+
+    .promoText a {
+      margin-bottom: 20px;
+      color: black;
+      font-size: 35px;
+      text-decoration: none;
+    }
+
+    .collection a {
+      margin-top: 40px;
+      padding-bottom: 2px;
+      border-bottom: 0.8px solid black;
+      color: black;
+      font-size: 20px;
+      text-decoration: none;
+    }
+  }
+  .dualSectionWrap {
+    display: flex;
+    justify-content: center;
+    height: 620px;
+    margin-top: 100px;
+    margin-bottom: 60px;
+
+    .firstDualProduct {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: 620px;
+      height: 100%;
+      .productImgMed {
+        width: 627px;
+        height: 362px;
+      }
+      .showProduct {
+        margin-top: 40px;
+        text-align: center;
+        .productName {
+          display: inline-block;
+          padding-bottom: 10px;
+          color: black;
+          font-size: 20px;
+          text-decoration: none;
+        }
```

```suggestion
    .firstDualProduct {
      display: flex;
      flex-direction: column;
      justify-content: center;
      width: 620px;
      height: 100%;
      
      .productImgMed {
        width: 627px;
        height: 362px;
      }
      
      .showProduct {
        margin-top: 40px;
        text-align: center;
        
        .productName {
          display: inline-block;
          padding-bottom: 10px;
          color: black;
          font-size: 20px;
          text-decoration: none;
        }
```
스타일 적용한 부분은 가독성을 위해서 한칸씩 띄워주세요!